src/seedu/addressbook/ui/Formatter.java

```diff
@@ -74,7 +74,7 @@ private static String getIndexedListItem(int visibleIndex, String listItem) {
     public static String getPrintableString(boolean showPrivate, Printable... printables) {
         StringBuilder stringChain = new StringBuilder();
         for (Printable i: printables) {
-            stringChain.append(i.getPrintableString(showPrivate)).append(" ");
+            stringChain.append(i.getPrintableString(showPrivate) + "\n");
         }
```

Comment:
If you are using a string builder, then it is redundant to use the "+" concatenation, so it is better to chain append calls rather than using a concatenation in between.

Reply:
ok

Reply:
ok

Reply:
In the mean time, also resolve this

Reply:
Resolved

Reply:
You are still using the + concatenation rather than append calls
![image](https://user-images.githubusercontent.com/35750423/47544175-526b3400-d918-11e8-95ac-535ac53b08de.png)

Reply:
Yea, both that and this is from "File Changed"
![image](https://user-images.githubusercontent.com/35750423/47566550-b1ec3280-d95e-11e8-8cc3-1190abeed6c0.png)

I guess GitHub really messed up that day. I'll trust ya on this one, and merge it